### PR TITLE
Add operator+(double seconds) to TimeStampUTC

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by bska** [comment]
This doesn't need to be a member function and therefore shouldn't be.  Also, the argument should probably be a `std::chrono::seconds<double>` instead of a bare `double`.  Something along the lines of
```C++
TimeStampUTC operator+(const TimeStampUTC& tp, const std::chrono::seconds<double> secs)
{
    return TimeStampUTC{ advance(asTimeT(tp), secs.count()) };
}
```
Then it's trivial to add `secs + tp` too.

**Comment by joakim-hove** [comment]
OK - fixed

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Ehh - no; not yeat. I just can't get this to link - any clues of what I am doing wrong:
```
/usr/bin/ld: CMakeFiles/TimeMapTest.dir/tests/parser/TimeMapTest.cpp.o: in function `TimeServiceOperatorPlus::test_method()':
/home/hove/work/OPM/opm-common/tests/parser/TimeMapTest.cpp:626: undefined reference to `Opm::operator+(Opm::TimeStampUTC, double)'
collect2: error: ld returned 1 exit status
```

NB: I intend to remove the double variatey; that was just be certain I was not fumbling with the std::chrono stuff.

File: src/opm/common/utility/TimeService.cpp [hunk 1]
```diff
@@ -117,3 +118,13 @@ std::time_t Opm::asTimeT(const TimeStampUTC& tp)
 
     return makeUTCTime(timePoint);
 }
+
+Opm::TimeStampUTC operator+(Opm::TimeStampUTC lhs, std::chrono::duration<double> delta) {
+    return Opm::TimeStampUTC( advance(Opm::asTimeT(lhs) , delta.count()) );
+}
+
```
Comment (bska) [inline src/opm/common/utility/TimeService.cpp]: The operator must be in namespace Opm:
```C++
Opm::TimestampUTC
Opm::operator+(...)
{
   /* ... */
}
```

Comment (joakim-hove) [reply src/opm/common/utility/TimeService.cpp]: Thanks!

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**